css/civicrm.css

```diff
@@ -2683,12 +2683,15 @@ div.crm-container form {
 /* Date plugin */
 .crm-container input.dateplugin,
 .crm-container input.crm-form-date {
-  background: white url(../i/cal.gif) no-repeat scroll right center;
-  padding-right: 16px;
-  /* so that text doesn't flow on top of icon */
   width: 9em;
 }
 
+.crm-container input.dateplugin::placeholder,
+.crm-container input.crm-form-date::placeholder {
+  font-family: "FontAwesome";
+  text-align: right;
```

Comment:
I believe you can also set the "content" attribute here so it's not needed with js.
(I always prefer css first if possible before resorting to js)

Reply:
That was my first instinct, too, but it doesn't work.  Or, rather when I tried it, it doesn't work, and apparently [it's webkit only](https://stackoverflow.com/questions/8075986/how-to-set-placeholder-value-using-css) and I was in Firefox.

Reply:
Ok, how about we do both for now, with a comment in the js that it's a FF shim that can be removed someday.

Reply:
Oops I should've read the comments over on SE--it doesn't work in Webkit anymore either.  JS it is.